### PR TITLE
OCPQE-14589: Configure ICSP on Hosted cluster

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Comment by LiangquanLi930** [comment]
```
oc debug -n kube-system node/ip-10-0-137-28.us-east-2.compute.internal                                           
Starting pod/ip-10-0-137-28us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`
Pod IP: 10.0.137.28
If you don't see a command prompt, try pressing enter.
sh-4.4# chroot /host
sh-4.4# cat /etc/containers/registries.conf
unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
short-name-mode = ""

[[registry]]
  prefix = ""
  location = "quay.io"
  mirror-by-digest-only = true

  [[registry.mirror]]
    location = "quayio-pull-through-cache-us-east-1-ci.apps.ci.l2s4.p1.openshiftapps.com"

[[registry]]
  prefix = ""
  location = "registry-proxy.engineering.redhat.com"
  mirror-by-digest-only = true

  [[registry.mirror]]
    location = "brew.registry.redhat.io"

[[registry]]
  prefix = ""
  location = "registry.redhat.io"
  mirror-by-digest-only = true

  [[registry.mirror]]
    location = "brew.registry.redhat.io"

[[registry]]
  prefix = ""
  location = "registry.stage.redhat.io"
  mirror-by-digest-only = true

  [[registry.mirror]]
    location = "brew.registry.redhat.io"
```

File: ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml [hunk 1]
```diff
@@ -48,6 +51,11 @@ chain:
       fi
       DOMAIN=${HYPERSHIFT_BASE_DOMAIN:-$DEFAULT_BASE_DOMAIN}
       RELEASE_IMAGE=${HYPERSHIFT_HC_RELEASE_IMAGE:-$RELEASE_IMAGE_LATEST}
+      
+      ICSP_COMMEND=""
```
Comment (csrwng) [inline ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml:55]: nit: s/ICSP_COMMEND/ICSP_COMMAND/ ?

**Comment by csrwng** [comment]
@LiangquanLi930 the change to hypershift-aws-create-chain looks good to me.  Just one nit.

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-ovn-hypershift-guest-p2-f14

**Pull request merged**